ahoopes16/tidkinbrae
# Initial Website

## Summary
This PR is being made to create a foundational Gatsby project to build the rest of my website on top of.

## Testing
1. Clone this repo
1. If you do not have the [Gatsby CLI](https://www.gatsbyjs.org/docs/gatsby-cli/) installed, run `npm install -g gatsby-cli`
1. Run `gatsby develop`
1. Visit http://localhost:8000 - you should see a very basic webpage
